pkgs/tools/networking/network-manager/default.nix

```diff
@@ -61,6 +61,10 @@ in stdenv.mkDerivation rec {
     # Meson does not support using different directories during build and
     # for installation like Autotools did with flags passed to make install.
     ./fix-install-paths.patch
+
+    # Fixes https://github.com/NixOS/nixpkgs/issues/72330
+    # Upstream MR: https://gitlab.freedesktop.org/NetworkManager/NetworkManager/merge_requests/323
+    ./0001-build-add-PPPD_PATH-to-config.h.meson.patch
```

Comment:
Could we fetch the [backported patch](https://gitlab.freedesktop.org/NetworkManager/NetworkManager/commit/4c11364201c094ad19ab9980ea6051a82bd2a550.patch), now that it has been merged?

Reply:
Was changing this before you've commented :smile:

Reply:
oop, wrong patch.